src/CBT.DotNetFx/packall.cmd

```diff
@@ -10,7 +10,7 @@ IF "%ProgramFiles(x86)%" NEQ "" (
 
 CALL :Net35
 
-FOR %%A IN (4.0.0,4.5.0,4.5.1,4.5.2,4.6.0,4.6.1,4.6.2) DO (
+FOR %%A IN (4.0.0,4.5.0,4.5.1,4.5.2,4.6.0,4.6.1,4.6.2,4.7.1,4.7.2) DO (
```

Comment:
This CBT.DotNetFx dir doesn't actually seem used AFAICT, but figured I'd get it up to date anyway. @jeffkl I defer to you as to whether this entire dir should be deleted instead.

Reply:
Yeah this is basically deprecated.  You can revert these files.  We didn't want to delete this yet in case we need to service any existing packages.

Reply:
It's not part of the sln and there is no reference to "packall" in the repo. Seems like this was an old way of creating these packages which predates each target framework version having its own project file.